### ISSUE TITLE
Question on Reconfiguring BattleOrders.js

### ISSUE THREAD

**Issue by Anabstudent** [kolton/d2bot-with-kolbot#3041]
TL;DR - I need help with a code question, is it possible to make your BO giver/getter wait per each WP by changing the BattleOrders.js script?

**Full story:**
Currently, my setup has 1 pally, 1 necro, and 6 sorceress. Basically I have them all telestomp together at all the MF spots and its been working pretty well for me. From time to time they separate, but for the most part they are together. The BO'ing does make the pally have to play catch up which can desync the characters slightly.

The pally is the BO giver and the others are the getters. At the start of the game the getters wait for the giver in the catacombs. Not including the one off instances where the other characters catch the BO, the getters do not get BO after the start of the game. My thought to make the BO more consistent and better clumping of the characters. Having reviewed the code, this all seems to check out.

What I do not know how is the follow - How could I modify the BattleOrders.js such that every time the giver/getters enter a new zone that they will wait for the BO cycle to be complete?

**Comment by gtoilet** [comment]
add delay to make them sit longer after each wp where the bo gets cast
the delay should be added in the other scripts
lets say you run dury and they have no bo at that time you add the delay in  after they take the wp for the getters script so they wait for the giver to finish

**Comment by Bcubrich** [comment]
Not exactly a solution, but might It might also be a good idea to give a few of your other charters CTA? I think they can still BO themselves if they have no BO if you set it up right, but I can't remember for sure.
